docs/TemplateManagementCLI.md

```diff
@@ -0,0 +1,101 @@
+Template Management is a tool to manage template files which is used in FHIR dotliquid converter engine. Template manager organizes template files in template image and allows pull and push operations. 
```

Comment:
Template CLI is a tool to manage template files for FHIR Converter DotLiquid engine. This CLI tool manages all template files in OCI image format and allows pushing and pulling templates from [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/).